src/CameraScreen/CameraKitCameraScreenBase.js

```diff
@@ -137,10 +169,10 @@ export default class CameraScreenBase extends Component {
 
   renderTopButtons() {
     return !this.props.hideControls && (
-        <SafeAreaView style={styles.topButtons}>
-            {this.renderFlashButton()}
-            {this.renderSwitchCameraButton()}
-        </SafeAreaView>
+      <SafeAreaView style={styles.topButtons}>
+        {this.renderFlashButton()}
+        {this.renderSwitchCameraButton()}
+      </SafeAreaView>
```

Comment:
What if SafeAreaView isn't available, for example, react-native v44, so you can do something like that:

```
import {SafeAreaView, View, ...} from 'react-native;
const Container = SafeAreaView && View;
```

Reply:
it's your code, but ok I will do it